alerting/src/main/kotlin/org/opensearch/alerting/AlertingPlugin.kt

```diff
@@ -320,6 +322,8 @@ internal class AlertingPlugin : PainlessExtension, ActionPlugin, ScriptPlugin, R
             AlertingSettings.ALERT_HISTORY_MAX_DOCS,
             AlertingSettings.ALERT_HISTORY_RETENTION_PERIOD,
             AlertingSettings.ALERTING_MAX_MONITORS,
+            AlertingSettings.PERCOLATE_QUERY_DOCS_SIZE_MEMORY_PERCENTAGE_LIMIT,
+            AlertingSettings.PERCOLATE_QUERY_MAX_NUM_DOCS_IN_MEMORY,
```

Comment:
do we rename this to `DOC_LEVEL_MONITORS` instead of `PERCOLATE_QUERY` as the concept of percolate query is internal to doc level monitors?

Reply:
renamed to `DOC_LEVEL_MONITOR_DOCS_SIZE_HEAP_PERCENTAGE_LIMIT`